src/utilities/CustomHooks.jsx

```diff
@@ -375,7 +378,7 @@ const usePlayListVideoActions = () => {
     }
   };
   const removeVideoFromPlaylist = async (videoId, playListId) => {
-    const encodedToken = localStorage.getItem("token");
+    const encodedToken = token();
     try {
       const response = await axios.delete(
```

Comment:
appreciation - I liked how you've implemented custom hooks, but since each hook function isn't reusable is it required to make it a custom hook ?

suggestion - I think you can put these functions inside the individual contexts & access them wherever needed. This is my suggestion, you can try your own implementation.

Reply:
Yeah the requirement for custom hooks comes with a reason since you can't access other hooks inside a function you need either functional component or the custom hook so there is the where I needed the dispatch from the useDataValues hook from the DataProvider there is the reason for this.And I don't get your point on not reusable since they are reusable where ever we need functionality .
Your suggestion is really nice of making context but I have not done so since I follow a rule when ever a variable or a function needs to have a global scope to use it any where I use context since I don't need these hooks and their functions more than 2 places so I avoided doing so.
Thanks for reviewing though aniket